src/workload.rs

```diff
@@ -308,11 +311,14 @@ impl LocalClient {
         // Currently, we just load the file once. In the future, we could dynamically reload.
         let data = tokio::fs::read_to_string(self.path).await?;
         let r: Vec<Workload> = serde_yaml::from_str(&data)?;
-        let mut wli = self.workloads.lock().unwrap();
-        for wl in r {
-            info!("inserting local workloads {wl}");
-            wli.insert(wl);
+        {
```

Comment:
want to know what's the purpose to new a block for workload insert for LocalClient?

Reply:
To drop the mutex

Reply:
I guess it might not be needed anymore, in initial iteration it was